internal/sdkv2provider/resource_cloudflare_record.go

```diff
@@ -431,12 +431,11 @@ func transformToCloudflareDNSData(recordType string, id string, value interface{
 	switch {
 	case id == "flags":
 		switch {
-		case strings.ToUpper(recordType) == "SRV",
-			strings.ToUpper(recordType) == "DNSKEY":
+		case strings.ToUpper(recordType) == "SRV":
 			newValue, err = value.(string), nil
 		case strings.ToUpper(recordType) == "NAPTR":
 			newValue, err = value.(string), nil
-		case strings.ToUpper(recordType) == "CAA":
+		case strings.ToUpper(recordType) == "CAA", strings.ToUpper(recordType) == "DNSKEY":
```

Comment:
can you please add test coverage for this change?

Reply:
Is there a good way to test DNSKEY records? I don't want to break DNSSEC on a zone

Reply:
the acceptance testing account is fine for this sort of test case if you can put it together, i (and CI) can run it.

Reply:
I think the test I added will work, but I need to get multi-provider DNSSEC enabled for the zone before I am able to run the test myself. I also used the public key for another of my zones, as I wasn't sure the best way to generate one on demand.